content/workshops/pki_workshop/unit_lessons/unit_11.md

```diff
@@ -0,0 +1,80 @@
+# Unit 11: Certificate profiles
+
+In this unit you will go over the caUserCert profile. This certificate profile is for enrolling user certificates.
+
+The CA profiles are located here:
+
+    /var/lib/pki/ca1/ca/profiles/ca
+
+The [caUserCert.cfg](https://gitlab.consulting.redhat.com/pki/pki-workshop/blob/master/unit_labs/rhcs/caUserCert.cfg) profile is located in the rhcs labs folder.
+
+## Example caUserCert Profile
+
+The first part of a certificate profile is the description. This shows the name, long description, whether it is enabled, and who enabled it.
```

Comment:
The fields you will see in a CA User Profile are as follows:

- desc is a long description of the user certificate profile
- visible (true/false) controls whether the certificate will be visible to users
- enable, when selected (true), will allow the certificate to be used
- enableBy identifies the user who enabled the certificate
- name is the short name used for the certificate that will be viewable by users

Reply:
Fixed